src/main/java/org/motechproject/newebodac/web/JobSchedulerController.java

```diff
@@ -0,0 +1,58 @@
+package org.motechproject.newebodac.web;
+
+import java.util.List;
+import javax.validation.Valid;
+import org.motechproject.newebodac.dto.JobDto;
+import org.motechproject.newebodac.service.SchedulerService;
+import org.quartz.SchedulerException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+@Controller
+public class JobSchedulerController {
+
+  @Autowired
+  private SchedulerService schedulerService;
+
+  /**
+   * Triggers (reschedules) a single Quartz Scheduler job.
+   * @param jobDto a data transfer object of job to reschedule
+   * @return OK or BAD_REQUEST
+   * @throws SchedulerException throws exception in case any issues with scheduling
+   */
+  @PostMapping("/jobs")
```

Comment:
in all other endpoint we're using `@RequestMapping`, also I think the ulr should be 'job/trigger' or 'triggerJob'